cyto_dl/api/data.py

```diff
@@ -11,6 +11,7 @@ class ExperimentType(Enum):
     LABEL_FREE = "labelfree"
     SEGMENTATION_PLUGIN = "segmentation_plugin"
     SEGMENTATION = "segmentation"
+    SEGMENTATION_ARRAY = "segmentation_array"
```

Comment:
would be helpful to make these options more visible. Maybe in readme?

Reply:
yes, I can also add a function to show available options